lib/diagnostic.rb

```diff
@@ -2,25 +2,25 @@
 Diagnostic = OpenStruct.new
 
 # Running scripts from the command line
-#
+
 # In a Ruby string, write the command you would use to run a script at
 # `lib/example.rb`
 
-Diagnostic.script = 'your response here'
+Diagnostic.script = 'Ruby lib/example.rb'
 
 # Using the REPL
 #
 # In a Ruby string, write the command you would use to enter the REPL we
 # prefer over `irb`
 
-Diagnostic.repl = 'your response here'
+Diagnostic.repl = 'pry'
 
 # Naming conventions in Ruby
 #
 # In a Ruby string, write what you would write to properly name a variable in
 # Ruby that read in English as "star wars episode viii".
 
-Diagnostic.variable = 'your response here'
+Diagnostic.variable = 'star_wars = star wars episode viii'
```

Comment:
That's how to assign a variable, although that wouldn't work. 

Just looking for camel case, which you had in your variable name.